src/App.tsx

```diff
@@ -3,29 +3,70 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { Todo } from './react-app-env.d';
+import { getTodos } from './api';
 
-interface State {
-  selectedUserId: number;
-}
+type State = {
+  selectedUserId: number,
+  todos: Todo[],
+};
 
 class App extends React.Component<{}, State> {
   state: State = {
     selectedUserId: 0,
+    todos: [],
```

Comment:
You have `todos` array in App and `TodoList` component, it's unnecessary to store this array in two places. Let's optimize it and move handler `changeTodoStatus()` to the `TodoList` and then remove todos array from `App's` state.